abdm_python_integrator/abha/urls.py

```diff
@@ -0,0 +1,27 @@
+from django.urls import path
+
+from abdm_python_integrator.abha.views import abha_creation_views, abha_verification_views
+
+abha_creation_urls = [
+    path('api/generate_aadhaar_otp', abha_creation_views.GenerateAadhaarOTP.as_view(),
```

Comment:
Have you thought about using the ABHA_URLs verbatim here? It helps developers since it's just the ABHA documented URL (and know that it works in HQ) and more importantly allows for versioned endpoints. For e.g. we would be in difficultly when there will be v2 of the ABHA API that we need to upgrade to.

(I do see that the naming scheme is inconsistent with out naming scheme)

Reply:
Good Point. Agree that almost all the package M1 APIs are just wrappers over the ABDM ones.
One point to note is a couple of APIs have additional functionality linked to them. For e.g. `verify_mobile_otp` creates health id internally if verification is succesfull. _(Not the best way , think it was implemented based on mobile app requirement)_ .  Using the same verbatim for such cases wont depict the correct behaviour.

As for M2/M3 , there are couple of APIs that are actually wrapper over ABDM ones while others are not and serves other purposes. So I have refrained from using the same verbatim there in favour of consistency.
A couple of examples from M2 here -  follows the structure : `api/hiu/`
```
 path('api/hiu/generate_consent_request', GenerateConsent.as_view(), name='generate_consent_request'),
 path('api/hiu/consents', ConsentFetch.as_view({'get': 'list'}), name='consents_list'),
```
Probably we can consider the same structure for M1 as well `api/abha/` .

Reply:
That makes sense. I assume these are not planned for versioning for now (which is fine)